syntax  = "proto3";

message DroneData {
    float alt               = 1;
    float lat               = 2;
    float lon               = 3;
    repeated float speed    = 4;
    float airspeed          = 5;
    float pitch             = 6;
    float yaw               = 7;
    float roll              = 8;
    float bat               = 9;
    float vltg              = 10;
    float current           = 16;
    float temp              = 11;
    float distance          = 12;
    float groundspeed       = 13;
    string mode             = 14;
    bool armed              = 15;
    float home_lat          = 17;
    float home_lon          = 18;
}